### DISCUSSION TITLE
Ping status check

### DISCUSSION THREAD

**Discussion by M4rc311o** [Lissy93/dashy#1272] [Ideas]
I was reading Dashy *docs*, specifically the section about *status check*. There is mension about pinging a service, but it's not using actuall ICMP ping, only HTTP GET request. I have one device, which is not running any webserver, but I would love to have a tile in Dashy monitoring if this device is up with use just ICMP pings. Is this currently supported in some way? Thanks in advance.

**Comment by paulearley** [comment]
Adding a simple ping to  StatusCheck would be a GREAT addition. I have several apps that do not respond to web checks through http. 
Please consider adding this to an EXCELLENT addition to my work!
